xla/xla.proto

```diff
@@ -1098,7 +1098,11 @@ message DebugOptions {
   // be deterministic, although with additional overhead.
   bool xla_gpu_enable_scatter_determinism_expander = 345;
 
-  // Next id: 359
+  // Enable windowed einsum(collective matmul) rewrite for all-to-all + gemm
+  // This feature is still experimental.
```

Comment:
You should mention this only applies when the flag xla_gpu_multi_streamed_windowed_einsum is true, considering the flag (briefly) is still false by default.

Reply:
ah good point. updated